apps/assistants/views.py

```diff
@@ -65,7 +65,7 @@ class OpenAiAssistantTableView(SingleTableView, PermissionRequiredMixin):
     permission_required = "assistants.view_openaiassistant"
 
     def get_queryset(self):
-        return OpenAiAssistant.objects.filter(team=self.request.team).order_by("name")
+        return OpenAiAssistant.objects.filter(team=self.request.team, is_archived=False).order_by("name")
```

Comment:
You can use the [VersionsObjectManagerMixin](https://github.com/dimagi/open-chat-studio/blob/2030037d5e0ced3aafa06f3afe96fec18dec6c49/apps/experiments/models.py#L35C7-L35C33) for the `OpenAiAssistantManager` instead of doing this. This will update the default queryset to automatically exclude `is_archived=True` versions

Reply:
I apologize, a readme of sorts would probably have helped. It would be useful to know what we have for versioning models and what to keep in mind. I'll add one

Reply:
https://github.com/dimagi/open-chat-studio/pull/833/commits/382cca744d40f19cefe2489929c4fa89b153ce5a